cmd/diff-main.go

```diff
@@ -42,16 +42,12 @@ var diffCmd = cli.Command{
 	Flags:       append(diffFlags, globalFlags...),
 	CustomHelpTemplate: `NAME:
   {{.HelpName}} - {{.Usage}}
-
 USAGE:
   {{.HelpName}} [FLAGS] FIRST SECOND
 
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
-DESCRIPTION:
-  {{.Description}}
```

Comment:
@poornas, by removing description, we will lose this text:
`  Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. Objects of same name and size, but differ in contents are not noticed.` which is kind of important to know.

Can you you manage to add this information in USAGE ?